### ISSUE TITLE
build-time tests are not run on ci

### ISSUE THREAD

**Issue by madebr** [libsdl-org/SDL_image#379]
Looking into it atm.

Also, `testimage --filter Images` is currently failing.
The output contains a lot of:
```
ERROR: 08/01/23 17:26:46: Assert 'Loading reference should succeed (Parameter 'src' is invalid)': Failed
```

**Comment by slouken** [comment]
That sounds like the test data isn't where the program expects

**Comment by madebr** [comment]
Looks like only the xcf image decoder is failing:
```sh
ERROR: 08/01/23 23:25:18: Assert 'Surface differed from reference by at most 0 in 966 pixels': Failed
INFO: /home/maarten/projects/SDL_image/cmake-build-debug/test/sample.xcf:
```

**Comment by slouken** [comment]
Good catch, fixed in 698f2cf

**Comment by madebr** [comment]
Since https://github.com/libsdl-org/SDL_image/commit/ba43aa36fea179ae12586cb430cacc3e89ffe75c, the tests are run on ci.
